brainwave-prediction-app/GUI.py

```diff
@@ -132,19 +134,45 @@ def holding_pattern_window():
 # Define the layout for the Starting Page
 layout1 = [
     [sg.Button('Brainwave Reading', size=(20, 3)),
-     sg.Button('Transfer Data', size=(20, 3), disabled=True),
+     sg.Button('Transfer Data', size=(20, 3), disabled=False),
      sg.Button('Manual Drone Control', size=(20, 3)),
      sg.Button('Holding Pattern', size=(20, 3), disabled=True)]
 ]
 
-# Define the layout for the Transfer Data Page
-layout4 = [[sg.Button(
-    image_filename="/Users/williamdoyle/Documents/GitHub/Avatar/brainwave-prediction-app/images")]]
 
 # Create the windows
 window1 = sg.Window('Start Page', layout1, size=(1200, 800), finalize=True)
-window4 = sg.Window('Transfer Data', layout4, size=(
-    1200, 800), element_justification='c')
+
+
+# Transfer Window Screen
+def transfer_window():
+    # Define the layout of the transfer window
+    transfer_window_layout = [
+        [sg.Text("Insert a file")],
+        [sg.InputText(),
+         sg.FileBrowse("Browser File")],
+        [sg.Button('Transfer Data'), sg.Exit()],
+
+    ]
+ # Create a window with the defined layout
+    transfer_window = sg.Window(
+        "Transfer Data", transfer_window_layout, size=(1200, 800), element_justification='c')
+
+    while True:
+        # Read events and values from the window
+        event, values = transfer_window.read()
+
+        # Check if the window is closed or if the Exit button is clicked
+        if event in (sg.WIN_CLOSED, 'Exit'):
+            break
+
+        # Check if the Transfer Data button is clicked
+        if event == 'Transfer Data':
+            # If button is clicked send the transfer data to the 'file_transfer' file
+            print(values)
+            fileTransfer.transfer(None, None, values[0])

```

Comment:
This one is going to be more complicated to explain, but hear me out.

Currently, `fileTransfer` as a class is not a static and `transfer` is not a static method.  An object needs to be created.  So to correctly use the provided fileTransfer class, you should have somewhere in GUI.py `svrcon = fileTransfer()` and then you would do `svrcon.transfer(values[0], "This will be the target dir")`.

Ignoring that you are using None for self because an instance isn't being created, you have the `target `and `src` ordered incorrectly.  ` values[0]` is the src, so it should actually be the [0] argument.  I showed that in the above example.

Even in the `main()` of this file that is commented out you can see what I'm talking about.